enterprise/frontend/test/metabase-enterprise/sandboxes/sandboxes.cy.spec.js

```diff
@@ -772,5 +775,131 @@ describeWithToken("formatting > sandboxes", () => {
+            },
+            card_id: CARD_ID,
+            group_id: COLLECTION_GROUP_ID,
+            table_id: ORDERS_ID,
+          });
+        });
+        cy.log("**-- 2. Create the second native question with a filter --**");
+
+        cy.request("POST", "/api/card", {
+          name: "EE_520_Q2",
+          dataset_query: {
+            database: 1,
+            native: {
+              query:
+                "SELECT * FROM PRODUCTS↵WHERE CATEGORY={{sandbox}} AND PRICE>10",
+              "template-tags": {
+                sandbox: {
+                  "display-name": "Sandbox",
+                  id: "3d69ba99-7076-2252-30bd-0bb8810ba895",
+                  name: "sandbox",
+                  type: "text",
+                },
+              },
+            },
+            type: "native",
+          },
+          display: "table",
+          visualization_settings: {},
+        }).then(({ body: { id: CARD_ID } }) => {
+          cy.log(
+            "**-- 2a. Sandbox `Products` table based on this question --**",
+          );
+
+          cy.request("POST", "/api/mt/gtap", {
+            attribute_remappings: {
+              [ATTR_CAT]: ["variable", ["template-tag", "sandbox"]],
+            },
+            card_id: CARD_ID,
+            group_id: COLLECTION_GROUP_ID,
+            table_id: PRODUCTS_ID,
+          });
+        });
+
+        cy.log("**-- 3. Fetch permissions graph --**");
+
+        cy.request("GET", "/api/permissions/graph", {}).then(
+          ({ body: { groups, revision } }) => {
+            // Update permissions for `collections` group [id: 4]
+            // This mutates the original `groups` object => we'll pass it next to the `PUT` request
+            groups[COLLECTION_GROUP_ID] = {
+              1: {
+                schemas: {
+                  PUBLIC: {
+                    [PRODUCTS_ID]: { query: "segmented", read: "all" },
+                    [ORDERS_ID]: { query: "segmented", read: "all" },
+                  },
+                },
+              },
+            };
+
+            cy.log("**-- 4. Update/save permissions --**");
+
+            cy.request("PUT", "/api/permissions/graph", {
+              groups,
+              revision,
+            });
+          },
+        );
+
+        signOut();
+
+        cy.log("**-- Logging in as sandboxed user --**");
+        cy.request("POST", "/api/session", {
+          username: sandboxed_user.email,
+          password: sandboxed_user.password,
+        });
+      });
+
+      openOrdersTable();
+
+      // View details for Product ID #14
+      cy.get(".cellData")
+        .contains("14")
+        .click();
+      cy.findByText(/View details/i).click();
+
+      cy.log("**-- Reported failing on v1.36.x --**");
+      cy.log(
+        "**It should show object details instead of filtering by this Product ID**",
+      );
+      // The name of this product is visible in "details" only
+      cy.findByText("Awesome Concrete Shoes");
```

Comment:
The positive assertion needs a little adjustment - it should behave like simple sandboxing with "Filter by a column in the table":
- so that would mean it should show "Awesome Concrete Shoes" in the table
![image](https://user-images.githubusercontent.com/1447303/100472235-24024900-30dc-11eb-9812-72af07b6b83f.png)
- and clicking that FK->"View details" should take you to Object Detail, so the positive assertion could be Vendor="McClure-Lockman"
![image](https://user-images.githubusercontent.com/1447303/100472252-2fee0b00-30dc-11eb-84ac-5e8bff47f8eb.png)